wagtail/wagtaildocs/forms.py

```diff
@@ -12,3 +12,13 @@ def get_document_form(model):
             'tags': widgets.AdminTagWidget,
             'file': forms.FileInput()
         })
+
+
+def get_document_multi_form(model):
+    return modelform_factory(
+        model,
+        fields=['title', 'tags'],
```

Comment:
Now that we support custom document models, would it be worth making this configurable - either by re-using `model.admin_form_fields` minus `file`, or having a new attribute on AbstractDocument alongside `admin_form_fields`, just for this form?

(Also, setting a widget for `file` here is redundant)